vhdbuilder/packer/produce-packer-settings.sh

```diff
@@ -25,6 +25,7 @@ SUBSCRIPTION_ID="${SUBSCRIPTION_ID:-$(az account show -o json --query="id" | tr
 GALLERY_SUBSCRIPTION_ID="${GALLERY_SUBSCRIPTION_ID:-${SUBSCRIPTION_ID}}"
 CREATE_TIME="$(date +%s)"
 STORAGE_ACCOUNT_NAME="aksimages${CREATE_TIME}$RANDOM"
+E2E_LOCATION="${E2E_LOCATION:-westus3}"
```

Comment:
Maybe it's better to have a default and skip replication if the ENV variable isn't set?

Are we going to use it in multiple other pipelines? Maybe we don't need it every time.